api-testing/multicloud-api/src/main/java/com/soda/utils/Logger.java

```diff
@@ -0,0 +1,59 @@
+/*
+  Copyright 2020 The SODA Authors.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.soda.utils;
+
+/**
+ * This class used for logs print.
+ * TODO: Need to create generic method.
```

Comment:
Please open an issue for TODO

Reply:
This is repeated questions. Already discuss with @kumarashit.

Reply:
What is the conclusion of discussion, please update here.
Please Either open an issue or remove these lines

Reply:
I have created generic method. Please review